Install openshift python module failed in post installation steps

TASK [ocp-customization : Install openshift python module] FAILED!

Main error msg:
```
    creating build/temp.linux-ppc64le-3.6
    gcc -pthread -Wno-unused-result -Wsign-compare -DDYNAMIC_ANNOTATIONS_ENABLED=1 -DNDEBUG -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-protector-strong -grecord-gcc-switches -m64 -mcpu=power8 -mtune=power8 -funwind-tables -fstack-clash-protection -D_GNU_SOURCE -fPIC -fwrapv -fPIC -I/usr/include/python3.6m -c _ruamel_yaml.c -o build/temp.linux-ppc64le-3.6/_ruamel_yaml.o
    _ruamel_yaml.c:4:10: fatal error: Python.h: No such file or directory
     #include "Python.h"
              ^~~~~~~~~~
    compilation terminated.
```

--- a/playbooks/roles/ocp-customization/tasks/main.yaml
+++ b/playbooks/roles/ocp-customization/tasks/main.yaml
@@ -1,4 +1,12 @@
 # tasks file for ocp4 post install customizations
+
+- name: Install python devel
+  package:
+    name:
+      - python3
+      - python3-devel
+    state: present
+
 - name: Install openshift python module
   pip:
     executable: pip3

